xml/selinux.xml

```diff
@@ -467,97 +457,18 @@ system_u:object_r:var_t var</screen>
-   does not include a default or reference policy, and you must build a policy that is
-   customized for your installation.
- </para>
-   
- <!-- section removed, it is outdated for SLE 15*. cschroder 2021-7-14 -->  
- <!--
- <para>
-   For testing and learning, see The &selnx; Reference
-   Policy Project at <link xlink:href="https://github.com/SELinuxProject/refpolicy/wiki"/>.
-   You must have a policy, as &selnx; will not operate without one.
-  </para>--> 
-  <para>
-   After installing the policy, you are ready to start file system labeling.
-   Run
-  </para>  
-
-<screen>&prompt.sudo;restorecon -Rp /</screen>
-
+      
   <para>
-   to start the <command>/sbin/setfiles</command> command to label all files
-   on your system. The
-   <filename>/etc/selinux/minimum/contexts/files/file_contexts</filename>
-   input file is used. The <filename>file_contexts</filename> file needs to
-   match your actual file system as much as possible. Otherwise, it can lead
-   to a completely unbootable system. If that happens, modify the records in
-   <filename>file_contexts</filename> with the <command>semanage
-   fcontext</command> command to match the real structure of the file system
-   your server is using. For example
-  </para>
-
-<screen>&prompt.sudo;semanage fcontext -a -t samba_share_t /etc/example_file</screen>
-
-  <para>
-   changes the file type from the default <literal>etc_t</literal> to
-   <literal>samba_share_t</literal> and adds the following record to the
-   related <filename>file_contexts.local</filename> file:
-  </para>
-
-<screen>/etc/example_file unconfined_u:object_r:samba_share_t:s0</screen>
-
-  <para>
-   Then run
-  </para>
-
-<screen>&prompt.sudo;restorecon -v /etc/example_file</screen>
-
-  <para>
-   for the type change to take effect.
-  </para>
-
-  <para>
-   Before doing this, make sure to read the rest of this chapter, so you
-   fully understand how context type is applied to files and directories. Do
-   not forget to make a backup of the <filename>file_contexts</filename>
-   file before starting.
-  </para>
-
-  <note>
-   <title>The user <systemitem class="username">nobody</systemitem></title>
-   <para>
-    While using <command>semanage</command>, you may get a message that
-    complains about the home directory of
-    <systemitem class="username">nobody</systemitem>. In this case, change
-    the login shell of user <systemitem class="username">nobody</systemitem>
-    to <filename>/sbin/nologin</filename>. Then the settings of
-    <systemitem class="username">nobody</systemitem> match the current
-    policy settings.
-   </para>
-  </note>
-
-  <para>
-   After another reboot &selnx; should be operational. To verify this, use
-   the command <command>sestatus -v</command>. It should give you an output
-   similar to
+   Now you can reboot. System startup will take some time, as &selnx;
+   will relable the entire filesystem. After it is finished, run
```

Comment:
Will **what** the entire filesystem?

Should that be "re-label"?

Also, "file system".

Reply:
:+1: 

It's the Humpty Dumpty grammar. Words mean what I say.